hvplot/ui.py

```diff
@@ -540,16 +549,17 @@ def _plot(self, *events):
             feature_scale = kwargs.pop("feature_scale", None)
             kwargs['features'] = {feature: feature_scale for feature in kwargs.pop("features", [])}
         kwargs.pop('refresh_plot', None)
-
+        kwargs.pop('opts', None)
         kwargs['min_height'] = 600
+
         df = self._data
         if len(df) > MAX_ROWS and not (self.kind in KINDS["stats"] or kwargs.get('rasterize') or kwargs.get('datashade')):
             df = df.sample(n=MAX_ROWS)
         self._layout.loading = True
         try:
             self._hvplot = _hvPlot(df)(
                 kind=self.kind, x=self.x, y=y, by=self.by, groupby=self.groupby, **kwargs
-            )
+            ).opts(**opts_kwargs)
```

Comment:
Why not the options to the converter directly? hvPlot has some special options handling given the backend configuration https://hvplot.holoviz.org/user_guide/Plotting_Extensions.html#plot-styling-options

Reply:
It's less robust, and not all opts are available.